index.js

```diff
@@ -2120,24 +2124,41 @@
   //. Cons('red', Cons('blue', Cons('green', Nil)))
   //. ```
   function sortBy(f, foldable) {
-    var rs = reduce(function(xs, x) {
-      var fx = f(x);
-      var lower = 0;
-      var upper = xs.length;
-      while (lower < upper) {
-        var idx = Math.floor((lower + upper) / 2);
-        if (lte(xs[idx].fx, fx)) lower = idx + 1; else upper = idx;
-      }
-      xs.splice(lower, 0, {x: x, fx: fx});
+    var foldableIsArray = Array.isArray(foldable);
+    var idx = 0;
+    function reducer(xs, x) {
+      xs.push({idx: idx, x: x, prop: f(x)});
```

Comment:
Do you consider `prop` to be a better name than `fx` for the `f(x)` property?

Reply:
I'm indifferent. For `prop` I didn't actually look at the old name. The thought behind `prop` is that `f` returns some property of `x` (not property as in a JS property of an object but in the more general sense of the word). But I'd have no problem in switching it back.

Reply:
I prefer `fx`. :)

Reply:
:zap: